src/cli/commands/force/lightning/lwc/preview.ts

```diff
@@ -306,7 +306,7 @@ export class Preview extends SfdxCommand implements HasRequirements {
                       this.flags.apilevel
                   )
                 : new IOSEnvironmentRequirements(this.logger);
-            this._requirements.preview = {
```

Comment:
This was a bug that went unnoticed. Instead of assigning to `requirements` we were assigning to `this._requirements` which then was overwritten further down on line 316.